dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,44 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
+
+const args = { tags: ["alpha", "internal"], files: [] };
```

Comment:
what about beta for tags??

Reply:
alpha and internal are specified here as the base tags for when the tags are not specified. Beta can still be specified by providing the --tags arg followed by a comma seperated list of tags to look for.

Reply:
![nointernal](https://github.com/iTwin/eslint-plugin/assets/108944699/8ef76a37-65f4-4cbe-9e26-8e8353dcccd4)

Reply:
why was alpha included in the default? 
was that the previous behavior as well?

Reply:
Yes, that was the previous behavior if no tags were included with the command